src/components/ViewCourse/sections/ViewCourseReserves.js

```diff
@@ -215,7 +210,24 @@ const ViewCourseReserves = (props) => {
           );
         })
       }
-      {permissions.add && <AddReserve courseListingId={course.courseListingId} />}
+      { permissions.add && (
+        <Row>
+          <Col xs={12} md={6}>
+            <Card headerStart={<FormattedMessage id="ui-courses.addItem.existing" />}>
+              <div style={{ minHeight: '5rem' }}>
+                <AddReserve courseListingId={course.courseListingId} />
+              </div>
+            </Card>
+          </Col>
+          <Col xs={12} md={6}>
+            <Card headerStart={<FormattedMessage id="ui-courses.addItem.new" />}>
+              <Layout className="textCentered" style={{ marginTop: '2rem', minHeight: '5rem' }}>
```

Comment:
Again, not a review comment but a query: what does `<Layout>` actually _do_? (The documentation is sparse). How does this work differently than if you just used `<span>`?

Reply:
`Layout` gives us a bunch of Folio-standard CSS which we access by passing it `className`s. In this case, `Layout` has defined that `textCentered` means something and will apply it for us.

Reply:
Got it, thanks!